packages/vapor/scss/components/badge.scss

```diff
@@ -6,12 +6,11 @@
     padding: var(--badge-padding);
     color: var(--badge-text-color);
     font-weight: var(--badge-font-weight);
-    font-size: var(--badge-font-size);
     line-height: var(--badge-line-height);
     text-align: center;
-    text-transform: var(--badge-text-transform);
     background-color: var(--badge-background-color);
     border: var(--badge-border);
+    border-radius: 8px;
```

Comment:
Have a look at the `mod-rounded-border` util classes instead

Reply:
Also in the mockup, the enterprise badge actually has a border-radius of 6px

Reply:
The mockup shows border-radius as 8px for me in the Platform Light Reskin figma 👀 @oliviertaillon

Reply:
https://vapor.cloud.coveo.com/#/styles/borders

Reply:
Ahh interesting, the numbers for the utilities look like its half of the actual `border-radius` pixels, so in this case I would use the `mod-rounded-border-3`?